src/components/Publish/Pricing/index.module.css

```diff
@@ -2,6 +2,10 @@
   text-align: center;
 }
 
+.pricing ul > li {
```

Comment:
this seems like rather adventurous selector as it most likely matches way more than intended cause it matches all list items within the whole pricing screen. If there is no way to get a CSS module class onto what you want to select, then try to be as specific as possible, e.g. by using some attribute selector like we do in some places in this file like with `[class*='Tabs_tabContent']`